CHANGELOG.md

```diff
@@ -137,12 +137,12 @@ Color=#ccffcc
 
 ## [0.5] - 2018-09-16
 ### Added
-- wumgr.ini is restricted to be writeble only by administrators
-- added better download system, now server date is checked and files get downloaded only if there are newer ont he server
+- wumgr.ini is restricted to be writable only by administrators
+- added better download system, now server date is checked and files get downloaded only if there are newer on the server
 - automatic check for updates
 - added support url label
-- added customizable tools menu tin better integrate 3rd aprty tools, accessible from tray and the window system menu.
-- added update cache such to remembet the last state between application restarts
+- added customizable tools menu that better integrate 3rd aprty tools, accessible from tray and the window system menu.

```

Comment:
* `integrate` should be `integrates`
* `3rd aprty` should be `3rd-party` (hyphenated word + fix spelling of "party")